### DISCUSSION TITLE
Node SDK: streaming from microphone

### DISCUSSION THREAD

**Discussion by fillobotto** [deepgram/community#215] [General help]
Hi,
I followed [this guide](https://blog.deepgram.com/how-to-add-speech-recognition-to-your-react-project/)  to stream audio from microphone and convert it to text.
Unfortunaltely I only get empty responses from Deepgram SDK. I also tried downloading the audio stream locally to listen if the microphone actually records any audio input and indeed it does.

Any suggestion? Thanks

**Comment by jkroll-deepgram** [comment]
Hello, would you be able to share some additional details, such as the body of a request response from Deepgram, and/or a request ID?

You can also check out our [Live Streaming Audio Quickstart](https://developers.deepgram.com/docs/getting-started-with-live-streaming-audio) to see if there are any details or variations that could get you around this issue.

**Reply by fillobotto** [reply]
I'll provide them soon, however I'm running the same exact code you provided in the examples repository.
